tests/integration-tests/src/test/java/org/apache/activemq/artemis/tests/integration/amqp/JMSMessageConsumerTest.java

```diff
@@ -430,6 +430,15 @@ public void testSelectorsWithJMSDeliveryOnQueue() throws Exception {
       }
    }
 
+   @Test
```

Comment:
could a timeout be added?

Reply:
That test wasn't meant to stay in the PR as I just used it to reproduce the intermittent failure.  I've removed it and updated the PR.